test/unit/warp/src/Shfl.cpp

```diff
@@ -0,0 +1,110 @@
+/* Copyright 2021 David M. Rogers
```

Comment:
Please also add yourself and 2021 to the modified files

Reply:
I think this is done.